kratos/input_output/stl_io.cpp

```diff
@@ -17,17 +17,28 @@
 // Project includes
 #include "includes/define.h"
 #include "input_output/stl_io.h"
+#include "utilities/parallel_utilities.h"
 
 namespace Kratos
 {
 
-/// Constructor with  filenames.
-StlIO::StlIO(std::filesystem::path const& Filename)
-    : IO()
+StlIO::StlIO(std::filesystem::path const& Filename, const Flags Options)
+    : mOptions(Options)
 {
     std::fstream* pFile = new std::fstream();
```

Comment:
I just noticed that this class does manual memory management :'(
It doesn't have to be in this PR, but someone must absolutely replace `mpInputStream` with a `unique_ptr` .